src/ncp/spinel.h

```diff
@@ -28,9 +28,17 @@
 #ifndef SPINEL_HEADER_INCLUDED
 #define SPINEL_HEADER_INCLUDED 1
 
+#include "spinel_platform_config.h"
```

Comment:
Why would this need to be referenced here? Why not reference it only in `spinel.c`?